tutorials/rise_timeseries_frb.ipynb

```diff
@@ -0,0 +1,506 @@
+{
```

Comment:
<p>1 line explaining what that is?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dianna-ai/dianna/pull/714/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>It can be anything really, but I can mention a few examples</p><p> </p>

Reply:
<p>I see actually that the section I wrote on trying both the shift and a deleterious signal has disappeared :/ I'll rewrite that...</p>